content/tutorials/client-resources/index.md

```diff
@@ -94,6 +98,13 @@ When using the WebForms controls, there's an additional attribute, `HtmlAttribut
 <dnn:DnnJsInclude runat="server" FilePath="https://www.google.com/recaptcha/api.js" ForceProvider="DnnPageHeaderProvider" HtmlAttributesAsString="async:async,defer:defer" />
 ```
 
+Starting with DNN v9.11.0, `htmlAttributesAsString` can also be used in the SPA token to add custom html attributes to a script tag:
+```html
+[JavaScript:{ path: "/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributesAsString: "type:module"}]
+[JavaScript:{ path: "/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributesAsString: "nomodule:nomodule"}]
+[JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributesAsString: "async:async,defer:defer"}]
+```
+
```

Comment:
If https://github.com/dnnsoftware/Dnn.Platform/pull/5244 is merged as-is, the format of these tokens will change:

```suggestion
Starting with DNN v9.11.0, `htmlAttributes` can also be used in the SPA token to add custom html attributes to a script tag:
```html
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.esm.js", htmlAttributes: { type: "module" } }]
[JavaScript:{ path: "~/DesktopModules/ResourceManager/Scripts/dnn-resource-manager/dnn-resource-manager.js", htmlAttributes: { nomodule: "nomodule" } }]
[JavaScript:{ path: "https://www.google.com/recaptcha/api.js", htmlAttributes: { async: "async", defer: "defer" } }]
```

```